README.md

```diff
@@ -7,3 +8,129 @@
 A dataframe auditor that computes a number characteristics of the data. This is done by profiling the frame.
 
 
+
+> [Summary](#summary)
+> 
+> [Installation](#installation)
+>
+> [Testing](#testing)
+>
+> [Usage](#usage)
+> 
+> [Contributions](#contributions)
+
+## Summary
+
+  Data profiling is important in data analysis and analytics, as well as in determining characteristics of data pipelines.
+  This repository aims to provide a means to extract a selection of attributes from data.
+  
+  It is currently focused on processing _pandas_ dataframes, but this functionality is being 
```

Comment:
Specifically say python?

Reply:
It is clear from repo tags/display in github I feel. Also, pandas, so not going to fuss about it.

Reply:
cool